plugins/network/NetEng.hlf

```diff
@@ -44,30 +44,34 @@ $ #Network browser configuration#
  Enables adding "Network" item to the disks menu.
 
  #Add to plugins menu#
- Enables adding "Network" item to the plugins menu.
+ Enables adding "Network browser" item to the plugins menu.
 
- #Выводить ".." в списке провайдеров#
- Показывать ".." на панели в списке провайдеров.
+ #Show ".." in the providers#
```

Comment:
`the providers list` or `the list of providers`

Reply:
I took these lines (and below) from the NetEng.lng. But ok, once we started, let's put it in order.